lib/gds_design_system_breadcrumb_builder.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+# Custom BreadcrumbsOnRails::Breadcrumbs::Builder class
```

Comment:
Remove comments?

Reply:
these i put here deliberately as info. Arguably all classes should have high level descriptions to give context and example usage.